.circleci/src/workflows/prod_delivery.yml

```diff
@@ -7,18 +7,18 @@ jobs:
       requires:
         - check
         - bump_version
-  #  - build_ios: # TODO LUN-29
-  #      requires:
-  #        - check
-  #        - bump_version
+  - build_ios:
+      requires:
+        - check
+        - bump_version
   - deliver_android:
-      production_delivery: true # TODO test in mid of august
+      production_delivery: true # TODO test in mid of september
```

Comment:
What is meant by this comment? Aren't you on vacation then?

Reply:
No, I will try to deploy the app with the prod_delivery workflow after my vacations. This comment is just a reminder for me, that these workflow/jobs haven't not be tested and I need to do this.